lua/gitlab/utils/init.lua

```diff
@@ -122,7 +122,7 @@ M.time_since = function(date_string, current_date_table)
   local time_diff = current_date - date
 
   if time_diff < 60 then
-    return M.pluralize(time_diff, "second") .. " ago"
+    return "moments ago"
```

Comment:
This corresponds to what Gitlab shows:
```suggestion
    return "just now"
```